src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp

```diff
@@ -110,113 +252,75 @@ void BoundingVolumeHierarchyImpl<DeviceType>::queryDispatch(
   Kokkos::Profiling::pushRegion("ArborX:BVH:init_offset");
 
   reallocWithoutInitializing(offset, n_queries + 1);
-  Kokkos::deep_copy(offset, 0);
+  // Kokkos::deep_copy(offset, 0);
```

Comment:
Kept it because of https://github.com/arborx/ArborX/blob/d6d9ff1eed7f425b866d979b017a9b09cebb5bcc/src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp#L403-L404